### PR TITLE
[Merged by Bors] - feat(Order/RelIso/Basic): `RelIso.compl`

### PR DESCRIPTION
**Pull request by vihdzp**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge

File: Mathlib/Order/RelIso/Basic.lean [hunk 1]
```diff
@@ -635,9 +635,13 @@ protected theorem cast_trans {α β γ : Type u} {r : α → α → Prop} {s : 
...
 
-/-- a relation isomorphism is also a relation isomorphism between dual relations. -/
+/-- A relation isomorphism is also a relation isomorphism between dual relations. -/
 protected def swap (f : r ≃r s) : swap r ≃r swap s :=
-  ⟨f.toEquiv, f.map_rel_iff⟩
+  ⟨f, f.map_rel_iff⟩
+
+/-- A relation isomorphism is also a relation isomorphism between complemented relations. -/
+protected def compl (f : r ≃r s) : rᶜ ≃r sᶜ :=
+  ⟨f, not_iff_not.2 f.map_rel_iff⟩
```
Comment (YaelDillies) [inline Mathlib/Order/RelIso/Basic.lean:644]:
```suggestion
  ⟨f, f.map_rel_iff.not⟩
```

File: Mathlib/Order/RelIso/Basic.lean [hunk 2]
```diff
@@ -635,9 +635,13 @@ protected theorem cast_trans {α β γ : Type u} {r : α → α → Prop} {s : 
...
   ext fun x => by subst h₁; rfl
 
-/-- a relation isomorphism is also a relation isomorphism between dual relations. -/
+/-- A relation isomorphism is also a relation isomorphism between dual relations. -/
 protected def swap (f : r ≃r s) : swap r ≃r swap s :=
-  ⟨f.toEquiv, f.map_rel_iff⟩
+  ⟨f, f.map_rel_iff⟩
+
+/-- A relation isomorphism is also a relation isomorphism between complemented relations. -/
+protected def compl (f : r ≃r s) : rᶜ ≃r sᶜ :=
```
Comment (YaelDillies) [inline Mathlib/Order/RelIso/Basic.lean:643]:
```suggestion
@[simps]
protected def compl (f : r ≃r s) : rᶜ ≃r sᶜ :=
```

**Comment by YaelDillies** [comment]
maintainer merge

**Pull request closed without merging**